bower.json

```diff
@@ -1,7 +1,7 @@
 {
   "author": "Mozilla",
   "name": "marketplace-core-modules",
-  "version": "1.1.0",
+  "version": "1.2.0",
```

Comment:
@ngokevin: did you say it was convention to prepend the version number to a commit message?